client/tests/integration/tx_chainid.rs

```diff
@@ -0,0 +1,64 @@
+use std::str::FromStr;
+
+use iroha_crypto::KeyPair;
+use iroha_data_model::prelude::*;
+use test_network::*;
+
+use crate::integration::asset::asset_id_new;
+
+#[test]
+fn send_tx_with_different_chain_id() {
+    let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(11_240).start_with_runtime();
+    wait_for_genesis_committed(&[test_client.clone()], 0);
+    // Given
+    let sender_account_id = AccountId::from_str("sender@wonderland").expect("Valid");
```

Comment:
```suggestion
    let sender_account_id = AccountId::from_str("sender@wonderland").unwrap();
```

Please, write like so in a new code

Reply:
Same for other places in this file